src/component/user/show-user-component.jsx

```diff
@@ -26,19 +26,15 @@ export default class ShowUserComponent extends React.Component {
     }
 
     updateLocale() {
-        const locale = this.props.location
-            ? this.props.location.locale
-            : this.possibleLocales[this.possibleLocales.length - 1];
+        const locale = (this.props.location && this.props.location.locale) || navigator.language;
```

Comment:
This will actually not work for IE11 ([see details](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language)), where we have to use `navigator.userLanguage`. 

Would you mind creating a function for extracting the default languages from the browser which first checks `navigator.language` and then `navigator.userLanguage` if that does not exists?

Reply:
I made the change, but it doesn't work on IE11 anyway because there are methods that need polyfills, e.g. Array.prototype.find and String.prototype.startsWith.